src/DataModel/IData.ts

```diff
@@ -0,0 +1,6 @@
+import {IMetadata} from "./IMetadata";
+
+export interface IData extends Object
```

Comment:
Bundling hypermedia controls into a property seems like a clean approach. 👍  But what are they, exactly? Affordances? Operations? Actions? Controls? [Naming things is hard](https://twitter.com/secretGeek/status/7269997868).